src/Render.cc

```diff
@@ -57,13 +64,26 @@ namespace drafter {
             case JSONRenderFormat:
             {
                 refract::RenderJSONVisitor renderer;
-                refract::IElement* element = MSONToRefract(attributes);
+
+                refract::IElement* element = MSONToRefract(*attributes);
+
+                if (!element) {
+                    return body;
+                }
+
```

Comment:
No it is not clear. **if you did not allocate memory (or create instance of object), you never should not release it.** 
It is responsibility of context where it was created. This is typical one of reason why C programs failure on memory access.